public/main.js

```diff
@@ -0,0 +1,64 @@
+// get the map - set location 
+let mymap = L.map('map').setView([44.478, -73.211882], 16)
+L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    maxZoom: 19,
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+}).addTo(mymap)
+
+let path = window.location.pathname
+let pathArray = path.split('/')
+let id = pathArray.pop()
+
+let listContainer = document.getElementById('idList')
+
+async function getData(id) {
```

Comment:
the argument of id is never used within this function. You could always run the function with no argument